File: src/main/java/mods/orca/mffs/items/ModItems.java
package mods.orca.mffs.items;

import net.minecraft.entity.player.EntityPlayer;
import net.minecraft.item.Item;
import net.minecraft.item.ItemStack;
import net.minecraft.nbt.NBTPrimitive;
import net.minecraft.nbt.NBTTagCompound;
import net.minecraft.util.ActionResult;
import net.minecraft.util.EnumHand;
import net.minecraft.world.World;

import java.util.ArrayList;
import java.util.Arrays;
import java.util.List;

public class ModItems {
    public static final List<Item> ITEMS = new ArrayList<>();

    public static final Item FREQCARD_BLANK = new ItemBase("freqcard_blank");
    public static final Item FREQCARD = new ItemFreqcard();
}

File: src/main/java/mods/orca/mffs/items/ModItems.kt
package mods.orca.mffs.items

import net.minecraft.item.Item

object ModItems {

    @JvmField
    val ITEMS: MutableList<Item> = ArrayList()

    @JvmField
    val FREQCARD_BLANK = ItemBase("freqcard_blank")

    @JvmField
    val FREQCARD = ItemFreqcard()

}
